server/src/com/cloud/user/AccountManagerImpl.java

```diff
@@ -2221,6 +2222,24 @@ private UserAccount getUserAccount(String username, String password, Long domain
     }
 
     @Override
+    public List<String> getKeys(GetUserKeysCmd cmd){
```

Comment:
Instead of a list should this be a Pair (keypair) of string, and rename the method as getUserKeyPair() ?